hail/Makefile

```diff
@@ -294,7 +294,7 @@ docs: install $(PYTHON_VERSION_INFO)
 	mkdir -p build/www/docs
 	rm -rf build/www/docs/$(HAIL_MAJOR_MINOR_VERSION)
 	mv build/docs/_build/html build/www/docs/$(HAIL_MAJOR_MINOR_VERSION)
-	find ./build/www -iname *.html -type f -exec sed -i '' -e "s/\.css/\.css\?v\=$(HAIL_CACHE_VERSION)/" {} +;
+	find ./build/www -iname *.html -type f -exec sed -i -e "s/\.css/\.css\?v\=$(HAIL_CACHE_VERSION)/" {} +;
```

Comment:
I was getting errors when building the docs with file not found `` (empty string). This change brings the makefile in line with the build yaml [here](https://github.com/chrisvittal/hail/blob/9335c61b4e200c5fe7b9cdd34e5efcd49c2aa7e6/build.yaml#L516), while removing the error.